src/components/ScreenWrapper/index.native.js

```diff
@@ -0,0 +1,24 @@
+import {React, useEffect} from 'react';
+import DevMenu from 'react-native-dev-menu';
+import toggleTestToolsModal from '../../libs/actions/TestTool';
+import {propTypes, defaultProps} from './propTypes';
+import BaseScreenWrapper from './BaseScreenWrapper';
+
+const ScreenWrapper = (props) => {
+    useEffect(() => {
+        DevMenu.addItem('Open Test Preferences', toggleTestToolsModal);
+    }, []);
```

Comment:
I feel uncomfortable with creating a platform specific implementation for this entire ScreenWrapper component. Instead, could you please create a CustomDevMenu component which is rendered inside the Screen Wrapper? On native it will implement this effect and render nothing, on other platforms it would be a noop `() => {}`.

Sorry to come in late with this suggestion, but it will make the changes in this PR much smaller and I think it's a better plan. Sound good?

Reply:
Yeah I totally agree and that's a great idea @neil-marcellini. Sorry we didn't put this earlier @alexxxwork.